kong/plugins/jwt/daos.lua

```diff
@@ -55,6 +56,7 @@ return {
                                                      "^PS256$",
                                                      "^PS384$",
                                                      "^PS512$",
+                                                     "^EdDSA$",
```

Comment:
This is not directly related to the EdDSA support, however I wonder why the pkey checks are not applied for ES256, ES384 and ES512. 

Wouldn't it make sense to apply this check for all algorithms and exclude the symmetric ones?
```
if_match = {
  not_match =  "^HS", 
},
```

Reply:
right, this list has grown over time and indeed it would make sense to apply pkey checks to all asymmetrical keys. I'll add this to my pile. Thanks for brining that up @27ascii